vmware-samples/vcd-ext-samples
# [VACE-2685] Create Dev Contaner in vmware-samples for 9.1, 9.5, 10.1

Create dev container env for 9.1.

Testing Done:
- Verify the env can be compiled.
- Verify the @vcd/sdk + @vcd/schematics + @vcd/plugin-builders are compatible with the env.
- Verify you can build a plugin using this env.
